cloud-aws-common/src/main/java/com/sequenceiq/cloudbreak/cloud/aws/common/AwsSetup.java

```diff
@@ -111,6 +111,10 @@ public void prerequisites(AuthenticatedContext ac, CloudStack stack, Persistence
     private void validateRegionAndZone(CloudCredential cloudCredential, Location location) {
         CloudRegions regions = awsPlatformResources.regions(cloudCredential, location.getRegion(), Collections.emptyMap(), true);
         List<AvailabilityZone> availabilityZones = regions.getCloudRegions().get(location.getRegion());
+        if (availabilityZones == null) {
```

Comment:
or empty

Reply:
Collections.isEmpty() ? 😸

Reply:
actually the empty is handled in the next lines nicely